modules/katello_repository_set.py

```diff
@@ -193,6 +197,7 @@ def main():
             state=dict(default='enabled', choices=['disabled', 'enabled']),
         ),
         supports_check_mode=True,
+        required_one_of=[['label', 'name']],
```

Comment:
Is product not required required, if label is missing?

Reply:
I was surprised also, notice the first one has a product, the second doesn't https://github.com/theforeman/foreman-ansible-modules/blob/master/test/test_playbooks/repository_set.yml#L7-L15

Reply:
I'd need to learn more about repo_sets to understand that...
Fine for merging.